lib/stitch_fix/log_weasel/logger.rb

```diff
@@ -2,6 +2,8 @@
 
 module StitchFix
   class LogWeasel::Logger < ::Logger
+    include LoggerSilence if defined?(LoggerSilence)
```

Comment:
@davetron5000 what say you on doing this instead?

Reply:
Ah, yeah, that makes sense (this way, the gem keeps working whenever Rails removes this module without telling us :)